webhooks/cluster.go

```diff
@@ -20,10 +20,14 @@ import (
 	"context"
 	"fmt"
 
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"sigs.k8s.io/cluster-api/feature"
+
```

Comment:
```suggestion
```

Reply:
I know we had this discussion before, but maybe we want to reconsider.

We could make this easier for reviewers and contributors by just enforcing the right order via the gci linter (supports make lint-fix).

The result would look like this: https://github.com/kubernetes-sigs/cluster-api/pull/5592

It's just that not linting it automatically but manually on PRs does not seem like an ideal solution.

Reply:
Yeah let's pick that one up